exam_4.py

```diff
@@ -0,0 +1,6 @@
+def sortString(str):
+    str=input("Bir kelime giriniz:")
+    return ''.join(sorted(str))

```

Comment:
Kod bu sekliyle cumlede gecen kelimelerin tamamini siralayip yazdiriyor. Soruda harfleri tekrar etmeyecek sekilde yazdirmaniz istenmisti.  Bunu `set()` ile saglayabilirsiniz:
```suggestion
    return ''.join(sorted(set(str)))
```